logging/logger.go

```diff
@@ -47,12 +48,6 @@ type ZapCompatibleLogger interface {
 	Fatalw(msg string, keysAndValues ...interface{})
 }
 
-// zLogger type for logging to. Wraps a zap logger and adds the `AsZap` method to satisfy the
```

Comment:
Didn't intend to move this. I just merge-conflicted poorly (I had this patch created prior to turning on all logging).